lib/jxl/encode.cc

```diff
@@ -1477,7 +1652,8 @@ JxlEncoderStatus JxlEncoderSetFrameName(JxlEncoderFrameSettings* frame_settings,
                                         const char* frame_name) {
   std::string str = frame_name ? frame_name : "";
   if (str.size() > 1071) {
-    return JXL_API_ERROR("frame name can be max 1071 bytes long");
+    return JXL_API_ERROR(frame_settings->enc, JXL_ENC_ERR_API_USAGE,
+                         "frame name can be max 1071 bytes long");
```

Comment:
perhaps mention what str.size() is?

Reply:
Sorry, missed this comment. Yes, could include that in the error message, but if you don't mind I'm not going to update this PR just for that since it's probably not _that_ important.